.github/workflows/test_in_devenv.yml

```
@@ -84,6 +89,48 @@ jobs:
               exit 1
             fi
 
+      - name: Test CUDA Quantum MPI4Py Plugin
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd $CUDAQ_REPO_ROOT
+            python3 -m pip install mpi4py~=3.1
+            rm -f build/lib/plugins/libcudaq-comm-plugin.so
+            ctest --test-dir build -R MPIApiTest -V
+            mpi4py_status=$?   
+            if [ ! $mpi4py_status -eq 0 ] ; then
+              echo "::error file=test_in_devenv.yml::Test CUDA Quantum MPI4Py Plugin failed with status $mpi4py_status."
+              exit 1
+            fi
+      
+      - name: Test CUDA Quantum MPI Plugin Activation
+        uses: ./.github/actions/run-in-docker
+        with:
+          image: cuda-quantum-dev:local
+          shell: bash
+          run: |
+            cd /usr/local/cudaq/distributed_interfaces/ 
```

```suggestion
            cd "$CUDAQ_INSTALL_PREFIX/distributed_interfaces/"
```
Better option instead: it would be nice if you just have activate_custom_mpi.sh cd into the directory (and back at the end) rather than forcing the one running the script to do that. This command works for bash (independent on how the script is invoked - sourced or otherwise) to get the directory that the script is in. I checked it on Windows and Linux, so I don't think it is OS specific, but I didn't check if it works with other shells than bash:
``` 
this_file_dir=`dirname "$(readlink -f "${BASH_SOURCE[0]}")"`
```